### PR TITLE
update linux install docs

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: desktop/faqs/linuxfaqs.md [hunk 1]
```diff
@@ -6,6 +6,165 @@ redirect_from:
...
+Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
+
+1. **To ensure that Docker Desktop provides a consistent experience across platforms**.
+
+    During research, the most frequently cited reason for users wanting Docker Desktop for Linux (DD4L) was to ensure a consistent Docker Desktop
+    experience with feature parity across all major operating systems. Utilizing
+    a VM ensures that the Docker Desktop experience for Linux users will closely
+    match that of Windows and macOS.
+
+    This need to deliver a consistent experience across all major OSs will become increasingly important as we look towards adding exciting new features, such as Docker Extensions, to Docker Desktop that will benefit users across all tiers.  We’ll provide more details on these at [DockerCon22](https://www.docker.com/dockercon/){: target="_blank" rel="noopener" class="_"}. Watch this space.
```
Comment (dvdksn) [inline desktop/faqs/linuxfaqs.md:98]: Let's reword this to make it future-proof! :-)

Comment (aevesdocker) [reply desktop/faqs/linuxfaqs.md:98]: Thank you for catching this @dvdksn!
